includes/auto_loaders/config.paypalwpp_incontext.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Autoloader array for PayPal Express In Context functionality. Makes sure that PayPalWPP In Context is instantiated at the
+ * right point of the Zen Cart initsystem.
+ * 
+ * @package     paypalwpp_incontext
+ * @author      mc12345678 
+ * @copyright   Copyright 2008-2016 mc12345678
+ * @copyright   Copyright 2003-2007 Zen Cart Development Team
+ * @copyright   Portions Copyright 2003 osCommerce
+ * @link        http://www.zen-cart.com/
+ * @license     http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version     $Id: config.paypalwpp_incontext.php xxxx 2016-08-20 11:15:10Z mc12345678 $
+ *
+ * Stock by Attributes 1.5.4
```

Comment:
I guess it stood out because the docblock segment is longer than the lines of code.

I'd remove `@link`. 

`@package` probably should probably be more consistent across all the files you're adding. Right now you've got `paypalwpp_incontext`, `paypalwpp`, `paymentMethod`, `page`.   

I'm inclined to suggest `paymentMethod`, or if you feel the distinction is beneficial then maybe `paypal_incontext` ... for all the files you're adding via this PR.